KS.Fiks.IO.Arkiv.Client/Schema/arkivmeldingOppdatering.xsd

```diff
@@ -0,0 +1,263 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns="http://www.arkivverket.no/standarder/noark5/arkivmeldingoppdatering/v2"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:n5mdk="http://www.arkivverket.no/standarder/noark5/metadatakatalog/v2"
+           xmlns:arkivmelding="http://www.arkivverket.no/standarder/noark5/arkivmelding/v2"
+           targetNamespace="http://www.arkivverket.no/standarder/noark5/arkivmeldingoppdatering/v2"
+           elementFormDefault="qualified">
+    <xs:import namespace="http://www.arkivverket.no/standarder/noark5/metadatakatalog/v2" schemaLocation="./metadatakatalog.xsd"/>
+    <xs:import namespace="http://www.arkivverket.no/standarder/noark5/arkivmelding/v2" schemaLocation="./arkivmelding.xsd"/>
+    
+    <xs:element name="arkivmeldingOppdatering" type="arkivmeldingOppdatering"/>
+
+    <xs:complexType name="arkivmeldingOppdatering">
+        <xs:sequence>
+            <xs:element name="meldingId" type="xs:string"/>
+            <xs:element name="tidspunkt" type="xs:dateTime"/>
+            <xs:choice minOccurs="1" maxOccurs="1">
+                <xs:element name="mappeOppdateringer" type="mappeOppdatering" minOccurs="0" maxOccurs="unbounded"/>
+                <xs:element name="registreringOppdateringer" type="registreringOppdatering" minOccurs="0" maxOccurs="unbounded"/>
+            </xs:choice>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="mappeOppdatering">
+        <xs:sequence>
+            <xs:choice minOccurs="1" maxOccurs="1"> <!-- Hvilke id'er er gyldige ved en oppdatering? Og burde de være i en choice sånn som dette med minOccurs=1? systemID, mappeID og referanseEksternNoekkel? Det må jo alltid være med 1 id og bare 1 av dem. -->
```

Comment:
Skal det være en choice her mellom systemID og mappeID?

Reply:
Nei, vi går for systemID så det blir konsistent.